### PR TITLE
release process fixes

### PR DESCRIPTION
**Pull request by alexcb**
- push to slack once homebrew branch is pushed
- check for overwritting existing release (and provide option to
confirm)
- seperate dind build target

Signed-off-by: Alex Couture-Beil <alex@earthly.dev>

### CODE REVIEW

File: release/README.md [hunk 1]
```diff
@@ -87,6 +87,15 @@ If the release-homebrew fails with a rejected git push, you may have to delete t
 
     git push "$GIT_USERNAME" --delete "release-$RELEASE_TAG"
 
+### dind
+
+Docker-in-Docker (dind) images change less frequently than earthly, but take a long time to build.
+These images can be rebuild by running:
```
Comment (vladaionescu) [inline release/README.md:93]:
```suggestion
These images can be rebuilt by running:
```

File: release/release.sh [hunk 2]
```diff
@@ -29,17 +29,28 @@ test -n "$RELEASE_TAG" || (echo "ERROR: RELEASE_TAG is not set" && exit 1);
...
 if [ -n "$GITHUB_SECRET_PATH" ]; then
     GITHUB_SECRET_PATH_BUILD_ARG="--build-arg GITHUB_SECRET_PATH=$GITHUB_SECRET_PATH"
+else
+    (../earthly secrets ls /earthly-technologies >/dev/null) || (echo "ERROR: current user does not have access to earthly-technologies shared secrets"; exit 1);
 fi
 
-../earthly upgrade
+(../earthly secrets get /user/earthly-technologies/aws/credentials >/dev/null) || (echo "ERROR: user-secrets /user/earthly-technologies/aws/credentials does not exist"; exit 1);
+
+# TODO validate the existing RELEASE_TAG does NOT exist, if it does prompt the user to ask if they are sure (or ask the user to set a RERELEASE_FORCE=1
```
Comment (vladaionescu) [inline release/release.sh:48]: Can remove this TODO now.

**Review by vladaionescu — APPROVED** [review]
:tada:

File: release/release.sh (hunk 2 above)
Comment (alexcb) [reply release/release.sh:48]: d'oh!

**Pull request merged**